### PR TITLE
Update: fix eslint indentations (fixes #1770)

### LINKED ISSUE #1770: Update: apply 4 space indentation
Fix indentation issues in the ESLint project
https://github.com/eslint/eslint/pull/1698#issuecomment-72916183

it is more update than actually a fix

<bountysource-plugin>

---
Want to back this issue? **[Place a bounty on it!](https://www.bountysource.com/issues/8340364-update-apply-4-space-indentation?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by valorkin**
Fix indentation issues in the ESLint project
https://github.com/eslint/eslint/pull/1698#issuecomment-72916183

it is more update than actually a fix

### CODE REVIEW

**Comment by valorkin** [comment]
@nzakas really small update enabling indent "4 spaces" for eslint
please review

File: tests/lib/config.js [hunk 1]
```diff
@@ -107,10 +107,10 @@ describe("Config", function() {
             assert.lengthOf(actual, 0);
         });
 
-         it("should return the path when a package.json file is found", function() {
+        it("should return the path when a package.json file is found", function () {
             var configHelper = new Config(),
                 expected = getFixturePath("broken", "package.json"),
-                // The first element of the array is the .eslintrc in the same directory.
+            // The first element of the array is the .eslintrc in the same directory.
```
Comment (lo1tuma) [inline tests/lib/config.js]: Why was this line changed?

Comment (valorkin) [inline tests/lib/config.js]: @lo1tuma I am to lazy, to format lines manually
so I formatted broken indent blocks automatically
can revert it if you want

it is not indent linter issue, if you asking about it

Comment (nzakas) [inline tests/lib/config.js]: Yeah, I think the comment needs to be aligned with the like underneath it.

**Comment by nzakas** [comment]
Aside from the comment, looks good. FYI that and we are good to go.

**Comment by nzakas** [comment]
To address concerns from the JSCS team, I've reverted the commit that added the indent rule. We'll need to come up with a different approach. Since this is related, we should hold off until we have a new indent rule.

**Comment by nzakas** [comment]
@valorkin can you rebase this onto master?

**Comment by nzakas** [comment]
Should work on top of existing master.

**Comment by lo1tuma** [comment]
@nzakas did you fix the [indentation of the comment](https://github.com/eslint/eslint/pull/1771#discussion_r24250064) during the merge?

**Comment by nzakas** [comment]
No, doing that now.

**Pull request merged**